packages/payload/src/admin/components/utilities/PathSegmentInfo/index.tsx

```diff
@@ -0,0 +1,27 @@
+export const getPathSegmentInfo = (pathSegments, segmentName) => {
```

Comment:
I don't like how this is written. It looks very fragile, what if somebody had custom routes?